README.md

```diff
@@ -1 +1,57 @@
-# Hello-word
\ No newline at end of file
+
+# Hello-word Project
+
+this is a simple project  microverse project that I did to help me to understand :
+How to Create branches with different base branches,
+ the concept of version control,
+and how linters are used in a real project.
+
+
+
+## Built With
+
+- Html
+- Css
+
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] Please make sure that you add the getting started section under this section and make it more descriptive. The content should be the steps to follow to get your project up and running on a local machine. 

  For example,
  ```
  You can add the steps to follow to clone your project,
  
  Step 1:if you don't have git installed on your machine you can download it from [here](https://git-scm.com/downloads).
  
  Once you have git installed on your machine you can clone your project by running the command below to clone your project to your local machine
  
  
  `git clone <your project link>`
  
  Alternatively, you can download the zip file of your project by clicking on the `Code` button on the right side of your project page and clicking on `Download ZIP`
  
  Step 2: Locate the folder where you cloned your project to and open the `index.html` file in your browser to view your project.
  ```